content/cloud/custom-domain.md

```diff
@@ -10,10 +10,33 @@ url = "https://github.com/fermyon/developer/blob/main/content/cloud/custom-domai
+
+To take advantage of Fermyon Cloud Custom Domain, you must delegate your domain to Fermyon Cloud. For a full set of instructions, please follow the [Ferymon Cloud Custom Domains tutorial](./custom-domains-tutorial.md). 
+
+If you do not own a domain (or are not interested in purchasing one), but would like to customize your Spin application's domain name while still keeping the `fermyon.app` apex domain, then Custom Fermyon Subdomains may be a good option for you. Check out the [Custom Fermyon Subdomains tutorial](./custom-fermyon-subdomain.md) for more instructions. 
+
 ### Terminology
 
+* Apex Domain: the `finickwhiskerse.com` part in `www.finickywhiskers.com`
+* Custom Domain: a personalized and unique address that individuals, businesses, or organizations can register and use for their workloads. In the context of Fermyon Cloud, this is the domain name you provide. 
+* Custom Fermyon Subdomain: a customized name you select following the pattern  `<custom-name>.fermyon.app`. 
+* Domain Name: human-readable address to identify a workload on the internet. For example, `finicky-whiskers-6cavavfh.fermyon.app` or `www.finickywhiskers.com`
+* Domain Name Service (DNS) provider: a service that manages and maintains a database of domain names and their correspoding IP addresses. 
+* Domain Registrar: a company or organization accredited by domain name registries to allow individuals and businesses to register and manage domain names for their websites. For example, [Namecheap](https://www.namecheap.com/) or [Google Domains](https://domains.google/?gad=1&gclid=Cj0KCQjwzdOlBhCNARIsAPMwjbyNg3a3GcrQjbs3vBSDg4AWnnEkc96mJupyg8nqqiKyxVCyaePRXfAaAjRLEALw_wcB&gclsrc=aw.ds) are domain registrars. 
+* Fermyon Domain Name: the default domain a Spin application gets allocated at creation time, which is a combindation of the application's name and a random string. For example, `finicky-whiskers-6cavavfh.fermyon.app`.
+* Subdomain: a hierarchical part of a larger domain name. It is added to the left of the primary domain name, separated by a dot (period). For example, `finicky-whiskers.fermyon.app` is a subdomain of `fermyon.app`
+
 ### Relationship Between Custom Domains And Spin Applications
 
+Currently there is a 1:1 relationship between a custom domain and a Spin application. This means you can apply a custom domain to at most one Spin application. Note that It can take up to 72 hours to delegate your domain. To move your custom domain between Spin applications, you must delete the custom domain and re-delegate it to Fermyon Cloud with the appropriate Spin application. We also do not support adding custom records at this time, aside from  Name Server (NS) records we automatically generate and a text (TXT) record if you have registered the domain with us previously. 
```

Comment:
A "NS record we automatically generate" can't be a "custom record" can it?  Maybe I don't understand what "We also do not support" means in this context - are you saying these are forbidden and will cause Fermyon Custom Domains to stop working?

Reply:
I am also not sure about this specific paragraph. @macolso can we get some more information about this please?

Reply:
I was trying to communicate that the user cannot add multiple records at this time. We allow the user to add a single record: the custom domain. So if they wanted to add both `www.coffee.com` and `coffee.com` to their Spin application, that wouldn't be possible. 

After the user provides their desired custom domain, we'll generate 4 nameserver records that the user must share with their domain registrar to delegate their domain to Fermyon cloud.